src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuildingWorker.java

```diff
@@ -324,9 +336,15 @@ public void onWorldTick(@NotNull final TickEvent.WorldTickEvent event)
 
         // If we have no active worker, grab one from the Colony
         // TODO Maybe the Colony should assign jobs out, instead?
-        if (!hasEnoughWorkers() && (getBuildingLevel() > 0 || this instanceof BuildingBuilder)
+        if (!hasEnoughWorkers()
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)